cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.tsx

```diff
@@ -99,7 +113,7 @@ const EdgeInfoContent: FC<{ selectedEdge: NonNullable<SelectedEdge> }> = ({ sele
                                         targetName={targetNode.name}
                                         targetType={targetNode.type}
                                         targetId={targetNode.objectId}
-                                        haslaps={!!targetNode.haslaps}
+                                        haslaps={!!targetNodeProperties?.haslaps}
```

Comment:
Might call this new prop `targetHaslaps` to keep it uniform with the other props?

Reply:
That would make sense but it is not a new prop. It was just missing the value for it. Prop `haslaps` is actually being used in a bunch of the HelpText files and in some others. Should I update all? Little nervous to make that many changes.
<img width="1326" alt="Screenshot 2024-08-08 at 12 18 43 PM" src="https://github.com/user-attachments/assets/b2df137a-2d2e-4ee3-b5fe-9439ebd64f67">

Reply:
That makes sense! We can leave it as is!